partcad/src/partcad/wrappers/wrapper_brep.py

```
@@ -0,0 +1,43 @@
+import wrapper_common
+from ..exception import BREPProcessingError
+
+
+def process(path):
+    try:
+        shape = TopoDS_Shape()
+        builder = BRep_Builder()
+        brep_tools = BRepTools()
+
+        if not brep_tools.Read_s(shape, path, builder):
+            raise BREPProcessingError(f"Failed to load BREP file: {path}")
```

Ruff (0.8.2)

Abstract `raise` to an inner function

(TRY301)